src/fvOptions/actuatorLineSource/actuatorLineElement/actuatorLineElement.C

```diff
@@ -817,12 +848,14 @@ void Foam::fv::actuatorLineElement::rotate
     if (rotateVelocity)
     {
         velocity_ = RM & velocity_;
+        chordRefDirection_ = RM & chordRefDirection_;
```

Comment:
This rotation should go in the same place where rotating the chord direction is, rather than in this `if` statement.

Reply:
I did that the first time, the problem in doing is that then this `rotate()` function is called when pitching the blades (around the `spanDirection`) and as a result the `chordRefDirection` is also pitched. On the other hand, if it is inside the `if statement`, when the rotation is called every iteration, the rotation is done only around the `axis` and then it behaves as expected.

Reply:
Ah, yes. Makes sense!

Reply:
We should add a comment here as to why `chordRefDirection` is only rotated when the velocity is rotated.